pkg/operator/ceph/csi/spec.go

```diff
@@ -133,6 +138,10 @@ const (
 	DefaultRBDLivenessMerticsPort    uint16 = 9080
 )
 
+const (
```

Comment:
can we be added to current const list here https://github.com/rook/rook/blob/master/pkg/operator/ceph/csi/spec.go#L102

Reply:
OK